bundle/src/main/java/com/adobe/acs/commons/wcm/impl/PropertyMergePostProcessor.java

```diff
@@ -21,183 +21,257 @@
 import java.util.Collection;
 import java.util.Date;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.LinkedHashSet;
 import java.util.List;
-import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.jackrabbit.JcrConstants;
 
 /**
  * ACS AEM Commons - Property Merge Sling POST Processor
  */
 @Component
 @Service
 public class PropertyMergePostProcessor implements SlingPostProcessor {
+
     private static final Logger log = LoggerFactory.getLogger(PropertyMergePostProcessor.class);
 
     private static final String AT_SUFFIX = "@PropertyMerge";
     private static final String ALLOW_DUPLICATES_SUFFIX = AT_SUFFIX + ".AllowDuplicates";
     private static final String TYPE_HINT_SUFFIX = AT_SUFFIX + ".TypeHint";
     private static final String IGNORE_PREFIX = ":";
+    private static final String ALL_TAGS = "merge-all-tags";
+    private static final String VALID_JCR_NAME = "[^:/\\[\\]\\|\\s*]+";
+    private static final Pattern VALID_TAG = Pattern.compile("^" + VALID_JCR_NAME + ":" + VALID_JCR_NAME + "(/" + VALID_JCR_NAME + ")*$");
```

Comment:
Selecting ONLY a tag name space is a valid tag value ... ex. 

`"^" + VALID_JCR_NAME + ":$"` is valid as well.